sodym/named_dim_arrays.py

```diff
@@ -385,42 +379,48 @@ def check_id0(self):
 
 
 class Flow(NamedDimArray):
-    """The values of Flow objects are the main computed outcome of the MFA system. A flow connects two processes. Its
-    name is set as a combination of the names of the two processes it connects.
+    """The values of Flow objects are the main computed outcome of the MFA system.
+    A Flow object connects two :py:class:`Process` objects.
+    The name of the Flow object is set as a combination of the names of the two processes it connects.
 
-    Note that it is a subclass of NamedDimArray, so most of the methods are defined in the NamedDimArray class.
-    """
+    Flow is a subclass of :py:class:`NamedDimArray`, so most of its methods are inherited.
 
-    from_process: Optional[Process] = None
-    to_process: Optional[Process] = None
-    from_process_name: str
-    to_process_name: str
+    **Example**
+
+    >>> from sodym import DimensionSet, Flow, Process
+    >>> goods = Dimension(name='Good', letter='g', items=['Car', 'Bus', 'Bicycle'])
+    >>> time = Dimension(name='Time', letter='t', items=[1990, 2000, 2010, 2020, 2030])
+    >>> dimensions = DimensionSet([goods, time])
+    >>> fabrication = Process(name='fabrication', id=2)
+    >>> use = Process(name='use', id=3)
+    >>> flow = Flow(from_process='fabrication', to_process='use', dims=dimensions)
+
+    In the above example, we did not pass any values when initialising the Flow instance,
+    and these would get filled with zeros.
+    See the validation (filling) method in :py:class:`NamedDimArray`.
+    """
+    model_config = ConfigDict(protected_namespaces=())
 
-    # TODO: Define flow by passing process objects, rather than from flow definition.
+    from_process: Process
+    to_process: Process
+    from_process_name: Optional[str] = None
+    to_process_name: Optional[str] = None
```

Comment:
from_process_name and to_process_name can be deleted imo? 
Also the validator check_process_names. 

Instead, we could make a class method from_definitions which does what currently initialize_flows in the MFASystem is doing.